### DISCUSSION TITLE
Support HEIF/HEIC through libvips

### DISCUSSION THREAD

**Discussion by dwightwatson** [vercel/vercel#4591] [Ideas]
As noted earlier (#1117) there was a request to add JPEG support with libvips. It would be great to see HEIF/HEIC support added as well - as it's becoming a more common format as a standard photo format from iPhones.

```json
"heif": {
  "id": "heif",
  "input": {
    "file": false,
    "buffer": false,
    "stream": false
  },
  "output": {
    "file": false,
    "buffer": false,
    "stream": false
  }
},
```

**Comment by styfle** [comment]
Hi @dwightwatson 

What is the json you shared here?

I believe we are using the latest libvips from https://github.com/libvips/libvips/releases
